tools/python.js

```diff
@@ -0,0 +1,242 @@
+#!/bin/sh
```

Comment:
Maybe calling it `python` would be better since it's not really a pure js file anyway?

Reply:
Fine with me.